Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -40,6 +41,17 @@ VolumeGC::VolumeGC(std::unique_ptr<BlobReader> reader) : m_reader(std::move(read
   };
 
   m_converted_banner = [this] { return LoadBannerFile(); };
+
+  constexpr u32 BTID_MAGIC = 0x44495442;
+  BootID triforce_header;
+  auto tmp_fs = std::make_unique<FileSystemGCWii>(this, PARTITION_NONE);
```

Comment:
You should use `GetFileSystem` here instead of manually creating a `FileSystemGCWii`, to ensure that the work done here can be reused later.